tests/test_interpolate.py

```diff
@@ -0,0 +1,227 @@
+from jax.config import config
+
+config.update("jax_enable_x64", True)
+
+from importlib import resources
+
+import numpy as np
+from rdkit import Chem
+
+from timemachine.fe import atom_mapping, interpolate, single_topology_v3
+from timemachine.fe.utils import get_romol_conf
+from timemachine.ff import Forcefield
+
+
+def test_align_harmonic_bond():
+    """
+    Test that we can align idxs and parameters correctly for harmonic bonds.
+    We expect that decoupled terms have their force constants turned set to zero,
+    while maintaining the same equilibrium bond lengths.
+    """
+    a, b, c, d, e, f, g, h, i, j = np.random.rand(10)
+
+    # tbd: what do we do if there are repeats?
+    # merge repeats into a single term first?
```

Comment:
For now, would it make sense to raise an error if there are repeats? (to prevent any potentially surprising behavior)

Reply:
It currently asserts

Reply:
> It currently asserts

nit: add test for this?

Reply:
willdo